app/editor/src/features/content/form/ContentSummaryForm.tsx

```diff
@@ -332,7 +345,7 @@ export const ContentSummaryForm: React.FC<IContentSummaryFormProps> = ({
             setFieldValue('timeTrackings', [
               ...values.timeTrackings,
               {
-                userId: userId ?? 0,
+                userId: userId,
```

Comment:
This can't ever be a `0`.  The page will redirect to `/error` if the user is missing.